rules/typescript.js

```diff
@@ -24,12 +24,12 @@ const typescriptRules = {
     2,
     {
       multiline: {
-        delimiter: 'comma',
+        delimiter: 'none',
```

Comment:
This wasn't being enforced prior, so adjusting to how we are actually formatting types.